### PR TITLE
testing PR

### PR DESCRIPTION
**Pull request by rafatislam1**

### CODE REVIEW

File: de-identification-app/src/test/java/com/ibm/whc/deid/app/endpoint/datamasking/DataMaskingControllerMaskingConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+import com.ibm.whc.deid.shared.pojo.config.ConfigSchemaType;
+import com.ibm.whc.deid.shared.pojo.masking.DataMaskingModel;
+
+@RunWith(SpringRunner.class)
+// force using a test profile to avoid using any other active profile
+// we do not have a real application-test.properties.
+@ActiveProfiles(profiles = { "test" })
+@AutoConfigureMockMvc
+@SpringBootTest(classes = Application.class)
+public class DataMaskingControllerMaskingConfiguration {
```
Comment (sfc42) [inline de-identification-app/src/test/java/com/ibm/whc/deid/app/endpoint/datamasking/DataMaskingControllerMaskingConfiguration.java]: Suggest this class name be DataMaskingControllerMaskingConfigurationTest instead.

**Review by sfc42 — CHANGES_REQUESTED** [review]

**Comment by goffr8** [comment]
Maybe just use "invalid" instead

**Comment by goffr8** [comment]
Maybe just use "invalid" for the invalid rule instead.

**Comment by goffr8** [comment]
> 
> 
> that is correct. This is an invalid masking file I am using to confirm that the "null" should return a 400 with a proper error message

Maybe just use "invalid" for the invalid rule instead.

**Review by sfc42 — APPROVED** [review]

**Pull request merged**